api/test/integration/env/base/haproxy-lb/Dockerfile

```diff
@@ -1,6 +1,6 @@
-FROM nginx
+FROM haproxytech/haproxy-ubuntu:2.7.8
```

Comment:
The latest patch is [2.7.11](https://hub.docker.com/r/haproxytech/haproxy-ubuntu), is there any reason why we are using `2.7.8` instead? We could consider using `2.7` as well.

Reply:
The reason why patch `2.7.8` was chosen is because it is the one used in other Wazuh components, such as [here](https://github.com/wazuh/wazuh-tools/blob/dev-wazuh-coordinator/framework/proxy.Dockerfile) in wazuh wazuh-tools. Perhaps using `2.7.11` is not a bad idea, but then we would have different versions across the various components. Would that be a good idea?

Reply:
Patches are only for fixes, with (usually) no behavior changes. Therefore, using 2.7.11 or 2.7 as Gasti suggested should have no negative impact. I agree with using the last available patch here.